internal/router/middlewares/ratelim_test.go

```diff
@@ -246,7 +92,7 @@ func TestRateLim10IPs(t *testing.T) {
 	// we never get a 429 status response. The request per second being done is
 	// clearly more than 10 per second, but from different addresses which should be fine.
 	for i := 0; i < 1000; i++ {
-		ctx := context.WithValue(context.Background(), ContextKeyAddress, strconv.Itoa(i))
+		ctx := context.WithValue(context.Background(), ContextIPAddress, strconv.Itoa(i))
```

Comment:
Based on the test description, `TestRateLim10IPs`, and the comments, I think this test might not have actually been working how we thought.
The context contains `ContextKeyAddress`, which is a *wallet address*.  I think this test was supposed to be testing with *IP address*.  
Regardless of what it was doing, it's now testing with IP address.

Reply:
Actually, the [IP Address comes from the HTTP request](https://github.com/tablelandnetwork/go-tableland/pull/460/files#diff-3fbceac783d96c5c5e7e85ccaeee8243d07d4faffff54e19c89ecd5415288181R33) and [not from the context as we did with the wallet address](https://github.com/tablelandnetwork/go-tableland/pull/460/files#diff-3fbceac783d96c5c5e7e85ccaeee8243d07d4faffff54e19c89ecd5415288181L45). This line is not actually needed (maybe it's better to remove it) for this test and is probably a leftover from copying code from other tests. 

And just for completeness, we do store the IP address in the context. That's why the variable `ContextIPAddress` exits. But that's for a different purpose (logging, and collecting metrics), and not rate limiting.